src/main/java/ch/ejpd/lgs/searchindex/client/entity/LandRegister.java

```diff
@@ -0,0 +1,24 @@
+package ch.ejpd.lgs.searchindex.client.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity(name = "land_register")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
why not @NoArgsConstructor(access = AccessLevel.PROTECTED) like the rest of the entities?
also it's the only entity which defines names with underscores ("land_register", "sender_id") - is that needed?